website/content/docs/getting-started.md

```diff
@@ -178,6 +178,8 @@ You can open Heron UI in your browser at [http://localhost:8889](http://localhos
 and see something like this upon successful submission of the topology:
 ![alt tag](/img/heron-ui.png)
 
+Note: ``heron-tracker`` (in Step 4) should be running at the same time so that Heron UI can work properly. 
```

Comment:
After Step 3, as ``heron-tracker`` is still running in the command line, the user might kill it by ``control + c``, so as to type the command to launch Heron Ui as suggested in Step 4. In such case, the ``heron-ui`` will not work properly and the users will be confused, because from the instructions alone they don't know heron-tracker should be running along with heron-ui. 

This is what exactly happened to me, when I tried to run Heron local mode step by step following the instructions. So I propose this PR, in hope that the misunderstanding will not happen on other users.